### DISCUSSION TITLE
PlayerEntityInteractEvent fired 2 times

### DISCUSSION THREAD

**Discussion by Distractic** [Minestom/Minestom#1647] [Help] [answered]
I'm listening  the `PlayerEntityInteractEvent` event.
When I use the right click on my entity, this is the log :

```
net.minestom.server.event.player.PlayerEntityInteractEvent@250ca051
MAIN
net.minestom.server.event.player.PlayerEntityInteractEvent@1dab93d9
OFF
```

I only use the right click, but the event is create two times, one for each hand
Is it an expected behavior ?

**Comment by ZakShearman** [comment, marked as answer]
Yes. It's fired for your main hand and off hand. You must filter it.
